index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+        <header>
+            <nav class="menu">
+                <h4 class="logo">My Logo</h4>        
```

Comment:
- [ ] i want to kindly suggest that rather than using an `hx` heading tag to display your logo you make use of an `a` tag, the reason for this change is because it is good for convention, semantics, and user experience(UX), [check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. :sparkles: :+1: :grin: 
- below is an example code that can be used to implement this change
```html
<a href="#">My Logo</a>
```
- [ ] As an addition since you do not have a logo yet you can consider putting your name in place of the `my logo` :+1: